app/controllers/submit.js

```diff
@@ -779,7 +797,8 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
                 });
             let tags = this.get('basicsTags').slice(0);
             Ember.A(tags);
-            tags.pushObject(tag);
+
+            tags.pushObject(fixSpecialChar(tag));
```

Comment:
This seems unnecessary. Special characters get messed up because of the server. User input that has not yet been to the server should not need to be altered/ "fixed". We should send the server exactly what the user intended to write.